recipes/jenkins.rb

```diff
@@ -249,8 +256,14 @@ def jenkins = Jenkins.getInstance()
       --domain #{server_name}
       --fullchain-file #{cert_path}
       --key-file #{key_path}
+      --reloadcmd /root/cert-update-hook.sh
+      --force
     )
-    not_if "test -d /root/.acme.sh/#{server_name}"
+    not_if {
+      # TODO the second guard clause can be removed after >= 0.6.0
+      File.directory?("/root/.acme.sh/#{server_name}") and
+      File.read("/root/.acme.sh/#{server_name}/#{server_name}.conf").match(/Le_ReloadCmd='__ACME_BASE64__START_c3lzdGVtY3RsIHJlc3RhcnQgbmdpbng=__ACME_BASE64__END_'/)
```

Comment:
In order to allow arbitrary script fields, acme.sh base64 encodes script lines. This is the acme base64 encoding of `/root/cert-update-hook.sh`